### ISSUE TITLE
else lint error on condition-begin-end

### ISSUE THREAD

**Issue by LuisUrrutia** [tsqllint/tsqllint#282]
The test suite is passing, but there is one case that isn't included, and it fails on that case.

replace the file `source/TSQLLint.Tests/UnitTests/LintingRules/conditional-begin-end/test-files/conditional-begin-end-no-error.sql` with this content (I added the else case):
```sql
IF(1 = 1)
BEGIN;
    SELECT 1
END;
ELSE
BEGIN;
  SELECT 2
END;
```

If u run the tests, it fails.

**Comment by nathan-boyd** [comment]
can this be closed?

**Comment by lowlydba** [comment]
Working properly now for me, thanks

**Comment by stephenweaver** [comment]
I think also ran across this and fixed in [PR](https://github.com/tsqllint/tsqllint/pull/291)
cc: @rz-1, @nathan-boyd

**Comment by ltylenda** [comment]
@nathan-boyd I don't see this as an issue anymore. I think it can be closed out by @stephenweaver 's PR https://github.com/tsqllint/tsqllint/pull/291
